src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -578,19 +715,16 @@ sub-<label>/[ses-<label>/]
     fmap/
         sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_epi.nii[.gz]
         sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_epi.json
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.nii[.gz]
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.json
 ```
 
-The phase-encoding polarity (PEpolar) technique combines two or more Spin Echo
-EPI scans with different phase encoding directions to estimate the underlying
-inhomogeneity/deformation map. Examples of tools using this kind of images are
-FSL TOPUP, AFNI 3dqwarp and SPM. In such a case, the phase encoding direction is
-specified in the corresponding JSON file as one of: `i`, `j`, `k`, `i-`, `j-`,
-`k-`. For these differentially phase encoded sequences, one also needs to
-specify the Total Readout Time defined as the time (in seconds) from the center
-of the first echo to the center of the last echo (aka "FSL definition" - see
-[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and
-[here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For
-example
+The phase-encoding polarity (PEpolar) technique combines two or more EPI scans, with different phase encoding directions to estimate the underlying inhomogeneity/deformation 
+map. In the case of arterial spin labeling, this usually concerns two ‘m0scans’. Examples of tools using this kind of images are FSL TOPUP, AFNI 3dqwarp and SPM. In such a case, 
+the phase encoding direction is specified in the corresponding JSON file as one of: `i`, `j`, `k`, `i-`, `j-`,`k-`. For these differentially phase encoded sequences, one also 
+needs to specify the Total Readout Time defined as the time (in seconds) from the center of the first echo to the center of the last echo (aka "FSL definition" - see [here]
+(https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and [here](https://lcni.uoregon.edu/kb-
+articles/kb-0003) how to calculate it). For example
```

Comment:
you have a line break in the middle of the link --> this breaks the markdown syntax.

Note how there is a line break between `...egon.edu/kb-` and `articles/kb-0003) ...`

```suggestion
(https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and [here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For example
```

Reply:
actually, also the `[here]` from the previous line needs to be on a single line with the following.

The markdown link syntax always needs to be on a single line: `[some text](url)` and NOT like this:

```
[this-link]
(http:www.wont-work.net)

[this-link](http:www.wont-work
-either.net)

[but-this-link](http:www.willwork.success)
```